examples/ps.py

```diff
@@ -0,0 +1,25 @@
+from ollama import ps, pull
+from ollama._types import ProcessResponse
+
+# Ensure at least one model is loaded
+response = pull('llama3.1', stream=True)
+progress_states = set()
+for progress in response:
+  if progress.get('status') in progress_states:
+    continue
+  progress_states.add(progress.get('status'))
+  print(progress.get('status'))
+
+print('\n')
+
+
+response: ProcessResponse = ps()
+for model in response.models:
```

Comment:
similar comment as list example

Reply:
Not sure what you mean - no details section in this response object

Reply:
I meant similar output format to list example